class-tgm-plugin-activation.php

```diff
@@ -2093,6 +2095,19 @@ public static function get_instance() {
 
 			return self::$instance;
 		}
+
+		/**
+		 * Adds css to admin head
+		 *
+		 * @since 2.6.2
+		 */
+		function tgm_css() {
+			echo '<style>
+			#tgmpa-plugins #the-list .required > th {
```

Comment:
Do we really need two IDs in the selector? At most, one should be needed, but ideally, none if classes are available.